health/micro-ui/web/micro-ui-internals/packages/modules/health-payments/src/pages/employee/EditRegister.js

```
@@ -0,0 +1,421 @@
+                    {renderLabelPair('HCM_AM_ATTENDANCE_OFFICER', individualsData?.Individual?.[0]?.name?.givenName)}
+                    {renderLabelPair('HCM_AM_ATTENDANCE_OFFICER_CONTACT_NUMBER', individualsData?.Individual?.[0]?.mobileNumber)}
+                    {renderLabelPair('HCM_AM_NO_OF_ATTENDEE', AttendanceData?.attendanceRegister[0]?.attendees?.length || 0)}
+                    {renderLabelPair('HCM_AM_CAMPAIGN_START_DATE', formatTimestampToDate(project?.[0]?.startDate))}
+                    {renderLabelPair('HCM_AM_CAMPAIGN_END_DATE', formatTimestampToDate(project?.[0]?.endDate))}
+                    {renderLabelPair('HCM_AM_EVENT_DURATION', attendanceDuration || 0)}
+                    {/* {renderLabelPair('HCM_AM_STATUS', t(data?.[0]?.musterRollStatus) || t('APPROVAL_PENDING'))} */}
+                </Card>
+                <Card className="bottom-gap-card-payment">
+                    <div className="card-heading" >
+                        <h2 className="card-heading-title"></h2>
+                        <Button
```

Biome (2.1.2)

[error] Provide screen reader accessible content when using heading  elements.

All headings on a page should have content that is accessible to screen readers.

(lint/a11y/useHeadingContent)